scripts/migrations/008_add_component_type_to_moderation_requests.py

```diff
@@ -87,7 +87,8 @@ def updateDocument(db, doc, newDict):
 for release in allReleases:
     document = release.value
     if PARENT_COMPONENT_ID in document.keys():
-        releaseData[release.id] = componentData[document[PARENT_COMPONENT_ID]]
+        if document[PARENT_COMPONENT_ID] in componentData.keys():
```

Comment:
I would join these two `if`s.
Also, I believe it is Python convention to write `if key in dict`, as opposed to `if key in dict.keys()`